src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2126,6 +2126,9 @@ protected override void BeginProcessing()
                     jobAssigned = jobObject.AssignProcessToJobObject(processInfo.Process);
                 }
 
+                // Fetch process handle to trigger Process object to update its state by calling SetProcessHandle, discard result.
```

Comment:
Would be good to link to the issue so people looking at the code can see why this process handle is retrieved.